test/e2e/redundancy/gateway_failover.go

```diff
@@ -94,9 +94,17 @@ func testGatewayFailOverScenario(f *subFramework.Framework) {
 	f.SetGatewayLabelOnNode(framework.ClusterA, initialGatewayNode.Name, false)
 
 	// Ensure the new engine pod is started before we run the connectivity tests to eliminate possible timing issue where,
-	// after deleting the old pod, we actually run the connectivity test against the oold engine instance before k8s has
+	// after deleting the old pod, we actually run the connectivity test against the old engine instance before k8s has
 	// a chance to react to stop the process/container etc.
-	newEnginePod := f.AwaitSubmarinerEnginePod(framework.ClusterA)
+	var newEnginePod *v1.Pod
+	for retries := 1; retries < 10; retries++ {
+		newEnginePod = f.AwaitSubmarinerEnginePod(framework.ClusterA)
+		if newEnginePod.Spec.NodeName == initialGatewayNode.Name {
+			time.Sleep(5)
+		} else {
+			break
+		}
+	}
```

Comment:
If the pod doesn't start for some reason then this will be a really long timeout 😄  We should be able to bake this into the underlying `AwaitUntil` call but that can be looked into later - let's get this in so we can get the tests running again.

Reply:
Yes :+1: I thought the same :)

Let me update the patch, I was using the web interface and forgot to add some imports